serial_linux.go

```diff
@@ -62,10 +62,26 @@ func openPort(name string, baud int) (rwc io.ReadWriteCloser, err error) {
 	}()
 
 	fd := f.Fd()
+	// set blocking / non-blocking read
+	var minBytesToRead uint8 = 1
+	if readTimeout > 0 {
```

Comment:
Can this logic that is duplicated twice be moved into a common shared function like

```
func posixTimeoutValues(readTimeout time.Duration) (vmin uint8, vtime uint8)
```